setup.py

```diff
@@ -844,6 +844,13 @@ def configure_extension_build():
             extra_compile_args += ['-g']
             extra_link_args += ['-g']
 
+    # special CUDA 11.7 package that requires installation of cuda runtime, cudnn and cublas
+    pytorch_extra_install_requirements = os.getenv('PYTORCH_EXTRA_INSTALL_REQUIREMENTS', '')
+    if pytorch_extra_install_requirements:
+        report(f"pytorch_extra_install_requirements: {pytorch_extra_install_requirements}")
+        extra_install_requires += pytorch_extra_install_requirements.split(",")
```

Comment:
Please use semicolon as separator
```suggestion
        extra_install_requires += pytorch_extra_install_requirements.split(";")
```